docs/building/unix-instructions.md

```diff
@@ -35,6 +36,7 @@ For example, for Ubuntu 16.x:
 sudo apt-get update
 sudo apt-get install git clang-3.9 cmake libunwind8 curl
 sudo apt-get install libssl1.0.0 libicu55
+sudo apt-get install libomp-dev
```

Comment:
Is this specific to all distros of Linux or just Ubuntu 18.04? If it is specific to just a given distro may be specify that distro as well.

Reply:
what about libiomp5 ? Mkl need it as well

Reply:
I just checked and both are installed because libomp5 is a dependency for libomp-dev.

Reply:
Good to know.